packages/lodestar/src/api/rest/logger/fastify.ts

```diff
@@ -0,0 +1,49 @@
+import {Stream} from "stream";
+import {IncomingMessage} from "http";
+import {FastifyRequest} from "fastify";
+import {ILogger} from "@chainsafe/lodestar-utils";
+
+export class FastifyLogger {
+
+  public readonly stream: Stream;
+
+  public readonly serializers = {
+    req: (request: IncomingMessage&FastifyRequest) => {
+      return {
+        msg: `Request: ${request.ip} -> ${request.hostname}\t${request.method}:${request.url}\tRequestId: ${request.id}`
+      };
+    },
+  };
+
+  private winston: ILogger;
```

Comment:
I think should be called something like `log` or `logger` since this is generic `ILogger`